lms/templates/main.html

```diff
@@ -182,3 +182,31 @@
     )
   ) if course_id and enrollment_action else ""
 }</%def>
+
+# Performance beacon for onload times
+% if settings.FEATURES.get('ENABLE_ONLOAD_BEACON', False):
+<script>
+  var sample_rate = ${settings.ONLOAD_BEACON_SAMPLE_RATE};
```

Comment:
Thanks for catching that.  I've updated the code.  

@alawibaba @benpatterson FYI